TeamCode/src/main/java/org/firstinspires/ftc/teamcode/rework/AutoTools/Movements.java

```diff
@@ -1,4 +1,4 @@
-package org.firstinspires.ftc.teamcode.rework.RobotTools;
+package org.firstinspires.ftc.teamcode.rework.AutoTools;
```

Comment:
Maybe rename the packages so they are lowercase?

Reply:
Not sure on this one. @samduong01?